client.go

```diff
@@ -1653,7 +1653,7 @@ func addMissingPort(addr string, isTLS bool) string {
 	if isTLS {
 		port = 443
 	}
-	return fmt.Sprintf("%s:%d", addr, port)
+	return net.JoinHostPort(addr, port)
```

Comment:
net.JoinHostPort(addr, strconv.Itoa(port))

Reply:
Sorry, yes, thank you - once again, having run tests this time.